src/main/java/frc/robot/subsystems/ball/feeder/FeederImpl.java

```diff
@@ -21,21 +21,25 @@ public FeederImpl(){
         super(1, TimeUnit.SECONDS);
         
         this.beltController = new WPI_TalonSRX(RobotMap.BELT_TALON_PORT);
-        this.hopperController = new WPI_TalonSRX(RobotMap.ROLLER_TALON_PORT);
+        this.hopperController = new WPI_TalonSRX(RobotMap.HOPPER_TALON_PORT);
         this.feederUltrasonic = new Ultrasonic(RobotMap.ULTRASONIC_PORTS[0], RobotMap.ULTRASONIC_PORTS[1]);
         
 
         this.poller = new UltrasonicPoller(feederUltrasonic, 3000, 300);
         this.distanceTuner = new SmartDashboardTuner("Ball Distance", 300, d -> this.poller.setDistance(d));
+        
+        this.hopperController.configContinuousCurrentLimit(10, 250);
+        this.hopperController.enableCurrentLimit(false);
     }
 
     public void enableConveyor() {
-        this.beltController.set(0.95d);
+        this.beltController.set(-0.95d);
         // TODO Find actual values
     }
 
     public void enableHopper() {
-        this.hopperController.set(-0.8d);
+        //this.hopperController.enableCurrentLimit(true);
```

Comment:
I think we can re-enable the current limiting logic here and in `stopHopper()` since we did find the source of the manual control bug at Durham, and this wasn't it.